src/test/java/com/browserstack/examples/config/WebDriverConfiguration.java

```diff
@@ -31,9 +30,11 @@
     @JsonProperty("remoteDriver")
     private RemoteDriverConfig remoteDriverConfig;
 
+
     public List<Platform> getActivePlatforms() {
         List<Platform> activePlatforms = Collections.emptyList();
         switch (driverType) {
+            case dockerDriver:
```

Comment:
Rename dockerDriver to onPremGridDriver.

Reply:
Renaming Done